app/views/howcanihelp/index.html.erb

```diff
@@ -0,0 +1,97 @@
+<% content_for(:head) do %>
+  <% browser_title = t(:browser_title_tag) %>
+  <title><%= browser_title %></title>
+  <%= javascript_include_tag 'blueimp_script' %> 
+  <meta property="og:title" content="How can I help other burners" />
+  <meta property="og:description" content="Helping other burners" />
+  <meta property="og:image" content="http://dreams.midburnerot.com/assets/coverphoto.png" />
+<% end %>
+
+<div class="top_message" dir=<%=t :html_direction %>>
```

Comment:
Since this is really similar to the camps index, I'd try to have some partials render these parts to avoid code duplication